benchmarks/automated/tests/suite/bert_cpu.yaml

```diff
@@ -10,7 +10,7 @@ bert:
             - 2
             - 4
             - 8
-        input: "https://raw.githubusercontent.com/pytorch/serve/master/examples/Huggingface_Transformers/Seq_classification_artifacts/sample_text.txt"
+        input: "./examples/Huggingface_Transformers/Seq_classification_artifacts/sample_text.txt"
```

Comment:
@lxning  - this input change will break the current dashboard automation since it uses 'wget' to fetch this file from the provided link.  Can you create a separate folder for these config, or modify the code to accept either a URL or a local file.